test/TestHelper.cpp

```diff
@@ -1113,6 +1233,8 @@ Options::Options(int argc, char** argv)
 		}
 		else if (arg == "--filltests")
 			fillTests = true;
+		else if (arg == "--fillchain")
+			fillBlockchain = true;
```

Comment:
Be consistent with names. If the param name is `--fillchain` name the variable `fillchain` (or `fillChain`, but I prefer exactly the same name).